proto/streamlit/proto/ForwardMsg.proto

```diff
@@ -72,6 +74,8 @@ message ForwardMsg {
     // for this one. If the client does not have the referenced message
     // in its cache, it can retrieve it from the server.
     string ref_hash = 11;
+
+    // Next: 13
```

Comment:
Should we also use `reserved` in `ForwardMsg`. I see a bunch of skipped numbers. Not crucial for this PR, just raising a point.

Reply:
I don't see any skipped numbers. Perhaps you're just confused about the location of my comment, above? I put it inside the wrong brackets (fixed now).